mne/utils/_logging.py

```diff
@@ -88,7 +90,6 @@ def verbose(function):
     """  # noqa: E501
     # See https://decorator.readthedocs.io/en/latest/tests.documentation.html
     # #dealing-with-third-party-decorators
-    from .docs import fill_doc
```

Comment:
@cbrnr this is where I just did `return function` to see how much time the `verbose` decorator was eating up. Turns out almost all of it was in `fill_doc` hence why I target it in this PR.